src/index.tsx

```diff
@@ -11,4 +11,16 @@ function start() {
   render(<Root />, document.getElementById("root"));
 }
 
-start();
+const prepare = async (): Promise<void> => {
+  if (process.env.VITE_APP_MOCK === "1") {
+    console.log("a");
```

Comment:
It'a good idea to print something as we are starting the "mock" mode but maybe we should show something more explicit like "Running with mock data, all HTTP calls will be intercepted by MSW" (if my understanding is correct?)

Spa

Reply:
@michaelrambeau My bad when I leave the log I used while developing on this PR. I'm going to remove it soon 😂😂😂😂 
`msw` itself already provides that log when we run `npm run start:mock`